src/app/window-handler.ts

```diff
@@ -89,9 +90,12 @@ export class WindowHandler {
         // Use these variables only on initial setup
         this.config = config.getConfigFields([ 'isCustomTitleBar', 'mainWinPos', 'minimizeOnClose', 'notificationSettings' ]);
         this.globalConfig = config.getGlobalConfigFields([ 'url', 'contextIsolation' ]);
+        const { customFlags } = config.getGlobalConfigFields([ 'customFlags' ]) as IConfig;
+        this.globalConfig.customFlags = customFlags;
 
         this.windows = {};
         this.contextIsolation = this.globalConfig.contextIsolation || false;
+        this.backgroundThrottling = !this.globalConfig.customFlags.disableThrottling;
```

Comment:
we can read `disableThrottling` directly from `customFlags`

Reply:
Addressed as part of the above comment.